examples/0_Getting_started.py

```diff
@@ -13,24 +13,23 @@
 #     name: python3
 # ---
 
-# # Getting Started
+# # Basic usage example
 
-# ## Introduction
-
-# This example describes how to connect to Granta MI and perform a basic query for impacted substances. It also
-# demonstrates how to view logging messages returned by the Granta MI server. For more details about the results of the
-# queries, see the examples in the [Impacted Substances](1_Impacted_Substances_Queries/index.rst) and
-# [Compliance](2_Compliance_Queries/index.rst) sections.
+# This example shows how to connect to Granta MI and perform a basic query for impacted substances. It also
+# demonstrates how to view logging messages returned by the Granta MI server. For more information about the
+#  results of the queries, see the examples in [Impacted Substances](1_Impacted_Substances_Queries/index.rst) and
+# [Compliance](2_Compliance_Queries/index.rst).
 
 # ## Connect to Granta MI
 
 # First, use the ``ansys.grantami.bomanalytics.Connection`` class to connect to the Granta MI server. The ``Connection``
-# class uses a fluent interface to build the connection, which is always invoked in the following sequence:
+# class uses a Fluent interface to build the connection, which is always invoked in the following sequence:
```

Comment:
@PipKat This is a lower case 'f', in this context fluent is an adjective, as opposed to the Ansys product (https://en.wikipedia.org/wiki/Fluent_interface).

```suggestion
# class uses a fluent interface to build the connection, which is always invoked in the following sequence:
```